### DISCUSSION TITLE
Difference between getgauge/cli and getgauge/gauge-cli

### DISCUSSION THREAD

**Discussion by pbanyard** [getgauge/gauge#2565] [General]
I'm trying to install and use gauge as part of an ADO pipeline, but having issues with the npm install. I can get this to work 
npm install -g @getgauge/gauge-cli but not the newer version npm install -g @getgauge/cli. This is behind a corporate firewall but proxy is set.

![image](https://github.com/getgauge/gauge/assets/29274108/67952742-8283-4a03-bb1c-d49c309e732e)

Any suggestions as to what is going on here?

Thanks

**Comment by zabil** [comment]
The error points to a networking issue. While downloading from github. Can you try using a command line similar to `wget` or `curl` in *nix to download the zip url? This way you can if the proxy settings are working.

**Reply by pbanyard** [reply]
Works fine using curl

![image](https://github.com/getgauge/gauge/assets/29274108/40e737a6-28e3-42c0-8be1-0d457c9b151c)

**Reply by zabil** [reply]
Does the same problem occur when running `npm install -g @getgauge/gauge-cli` in PowerShell (like the one used for curl)?

**Reply by pbanyard** [reply]
Same issue when run from PowerShell getgauge/gauge-cli works but the newer getgauge/cli doesn't:


![image](https://github.com/getgauge/gauge/assets/29274108/af0bcf3f-ed4a-48f7-8546-521a90e2989e)

**Reply by zabil** [reply]
`getgauge/gauge-cli` is deprecated and has not been updated. `getgauge/cli` is the right package.

This however works for me on my mac

```
getgauge/gauge-cli
```

<img width="419" alt="Screenshot 2024-07-02 at 14 28 15" src="https://github.com/getgauge/gauge/assets/54427/c47f7a4c-fe46-4414-b197-b33de6c45ef2">

Note this is without a proxy. Unfortunately I can't test this on windows + proxy.
